src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -467,12 +482,12 @@ public function getNodePathForIdentifier($uuid, $workspace = null)
 
         $this->assertLoggedIn();
 
-        $cacheKey = "nodes by uuid: $uuid, ".$this->workspaceName;
+        $cacheKey = "nodes by uuid: $uuid, $this->workspaceName";
```

Comment:
I think that works in higher PHP version it's my fault that I have bad version setup in PHPStorm. :(